addon/models/node.js

```diff
@@ -25,10 +35,12 @@ export default DS.Model.extend(OsfModel, {
         inverse: 'children'
     }),
     children: DS.hasMany('nodes', {
-        inverse: 'parent'
+        inverse: 'parent',
+        updateRequestType: 'POST'
     }),
     affiliatedInstitutions: DS.hasMany('institutions', {
-        inverse: 'nodes'
+        inverse: 'nodes',
+        serializer: relationshipSerializer('affiliatedInstitutions', 'institutions')
```

Comment:
Because children can use the node serializer, but institutions (and other relationships that use a `/relationships/model` ) require a specific (simpler) serialization of the relationship contents.